lib/absinthe/phase/schema/apply_declaration.ex

```diff
@@ -44,7 +44,8 @@ defmodule Absinthe.Phase.Schema.ApplyDeclaration do
         %{
           schema_definition
           | type_definitions:
-              Enum.map(schema_definition.type_definitions, &maybe_mark_root(&1, root_mappings))
+              Enum.map(schema_definition.type_definitions, &maybe_mark_root(&1, root_mappings)),
+            schema_declaration: declaration
```

Comment:
Storing the `schema_declaration` so it can be used by SDL rendering - it might contain directives that we want to render